engine/swarm/how-swarm-mode-works/pki.md

```diff
@@ -53,10 +52,52 @@ Certificate:
 ```
 
 By default, each node in the swarm renews its certificate every three months.
-You can run `docker swarm update --cert-expiry <TIME PERIOD>` to configure the
-frequency for nodes to renew their certificates. The minimum rotation value is 1
-hour. Refer to the [docker swarm update](../../reference/commandline/swarm_update.md)
-CLI reference.
+You can configure this interval by running the `docker swarm update
+--cert-expiry <TIME PERIOD>` command. The minimum rotation value is 1 hour.
+Refer to the
+[docker swarm update](/engine/reference/commandline/swarm_update.md) CLI
+reference for details.
+
+## Rotating the CA certificate
+
+In the event that a cluster CA key or a manager node is compromised, you can
+rotate the swarm root CA so that none of the nodes will trust certificates
+signed by the old root CA anymore.
+
+Run `docker swarm ca --rotate` to generate a new CA certificate and key. If you
+prefer, you can pass the `--ca-cert` and `--external-ca` flags to specify the
+root certificate and and to use a root CA external to the swarm. Alternately,
+you can pass the `--ca-cert` and `--ca-key` flags to specify the exact
+certificate and key you would like the swarm to use.
+
+When you issue the `docker swarm ca --rotate` command, the following things
+happen in sequence:
+
+1.  Docker generates a cross-signed certificate. This means that a version of
+    the new root CA certificate is signed with the old root CA certificate.
+    This cross-signed certificate is used as an intermediate certificate for all
+    new node certificates. This ensures that nodes that still trust the old root
+    CA will be able to validate a certificate signed by the new CA.
+
+2.  In Docker 17.06 and higher, Docker also tells all nodes to immediately
+    renew their TLS certificates. This process may take several minutes,
+    depending on the number of nodes in the swarm.
+
+    > **Note**: If your swarm has nodes with different Docker versions, the
+    > following two things are true:
+    > - Only a manager running Docker 17.06 or higher will tell nodes to renew
```

Comment:
Apologies, one more clarification - only a manager running docker 17.06 that is running as the leader will tell the nodes to renew.